siriuspy/siriuspy/devices/bpm.py

```diff
@@ -1207,19 +1237,20 @@ def set_switching_mode(self, mode='direct'):
         for bpm in self._devices:
             bpm.switching_mode = mode
 
-    def mturn_update_initial_orbit(self, consider_sum=False):
+    def mturn_update_initial_timestamps(self, consider_sum=False):
         """Call this method before acquisition to get orbit for comparison."""
-        self._initial_orbs = self.get_mturn_orbit(return_sum=consider_sum)
+        self._initial_timestamps = self.get_mturn_timestamps(
+            return_sum=consider_sum)
 
     def mturn_reset_flags(self):
         """Reset Multiturn flags to wait for a new orbit update."""
         for flag in self._mturn_flags.values():
             flag.clear()
 
-    def mturn_reset_flags_and_update_initial_orbit(self, consider_sum=False):
+    def mturn_reset_flags_and_update_initial_times(self, consider_sum=False):
```

Comment:
I think you could call this method: `mturn_reset_flags_and_update_initial_timestamps`.

Reply:
done!